src/encoded/static/libs/ui/node_graph.js

```diff
@@ -0,0 +1,254 @@
+const circlePlus = '\u2295';
+const circleMinus = '\u2296';
+
+const slowAnimation = 400;
+const fastAnimation = 0;
+
+const mobileLimit = 400;
+
+let offset = -40;
+let doubleOffset = -60;
+
+const ellipseSettings = [
+    { cx: 2, cy: 2, rx: 12, ry: 9 },
+    { cx: 2, cy: 2, rx: 6.5, ry: 6 },
+    { cx: -5, cy: 10, rx: 12, ry: 9 },
+    { cx: -5, cy: 10, rx: 6.5, ry: 5 },
+    { cx: 10, cy: 10, rx: 12, ry: 9 },
+    { cx: 10, cy: 10, rx: 6.5, ry: 5 },
+];
+
+// Standardizes node names
+function nodeKeyName(name) {
+    return name.replace(/\s/g, '').toLowerCase();
+}
+
+// Creates a curved (diagonal) path from parent to the child nodes
+function diagonal(d, s) {
+    const path = `M${d.x},${d.y}C${d.x},${(d.y + s.y) / 2} ${s.x},${(d.y + s.y) / 2} ${s.x},${s.y}`;
+    return path;
+}
+
+const drawTree = (d3, targetDiv, data, fullWidth, fullHeight, margin, selectedNodes, setSelectedNodes) => {
+    let textWrapWidth = 115;
+    let characterLimitForWrap = 18;
+    const isMobile = fullWidth < mobileLimit;
```

Comment:
With mobile-first design in mind, it may be better to check if page in fullWidth (desktop) rather than check if mobile. Plus, I assume fullWidth covers table in horizontal and vertical rotations.

If you decide to stay with fullWidth-first design mode, have you considered something like - 
https://stackoverflow.com/a/10364620/178550
https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia

where the css is read to determine what is mobile rather than Javascript having determining it